bundles/org.openhab.core/src/main/java/org/openhab/core/service/WatchQueueReader.java

```diff
@@ -269,75 +315,79 @@ public void run() {
 
                 key.reset();
             } catch (InterruptedException exc) {
-                logger.debug("WatchQueueReader interrupted; thread exiting: {}", exc.getLocalizedMessage());
+                logger.info("Caught InterruptedException. Shutting down.");
```

Comment:
This is expected to happen when the runtime is shutdown:

```suggestion
                logger.debug("Caught InterruptedException. Shutting down.");
```